.github/workflows/build-test-package.yml

```diff
@@ -238,11 +244,11 @@ jobs:
         path: Evaluated/ITKModuleTemplate/dist
 
   build-windows-python-packages:
-    runs-on: windows-2019
+    runs-on: windows-2022
     strategy:
       max-parallel: 2
       matrix:
-        python-version-minor: ["7", "8", "9", "10"]
+        python-version-minor: ["7", "8", "9", "10", "11"]
```

Comment:
It is kind of good that this module itself uses the verbose version of the CI specification, so the more complicated remotes have a template to follow during updating.

Reply:
CI for this module also requires the additional step of evaluating the cookiecutter template, so we won't be able to easily replace with a reusable workflow.

Reply:
Some other remote modules require extra stuff (e.g. Ultrasound requires many things, ome-ngff requires a few unusual build tools such as `perl` and `nasm`).